test-fixtures/update.py

```diff
@@ -0,0 +1,91 @@
+import os
+import re
+import sys
+import pathlib
+from shutil import copyfile
+import yaml
+
+
+import requests
+import semver
+from requests.exceptions import HTTPError
+
+
+BITRISE_STACK_INFO = 'https://app.bitrise.io/app/6c06d3a40422d10f/all_stack_info'
+'''
+curl ${BITRISE_STACK_INFO} | jq ' . | keys'
+[
+  "available_stacks",
+  "project_types_with_default_stacks",
+  "running_builds_on_private_cloud"
+]
+'''
+pattern = 'osx-xcode-'
+BITRISE_YML = 'bitrise.yml'
```

Comment:
I think this will need a path adjustment now.

Reply:
Changed to `'../bitrise.yml'`